content/kb/troubleshooting-volume-take-long-time-to-mount.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Troubleshooting: Longhorn volumes take a long time to finish mounting"
+author: Phan Le
+draft: false
+date: 2021-02-26
+catelogies:
+  - "csi"
+---
+
+## Applicable versions
+All Longhorn versions.
+
+## Symptoms
+
+When starting workload pod which uses Longhorn volumes,
+observe in Longhorn UI that the Longhorn volumes are attached quickly
+but it takes a long time for the volumes to finish mounting and workload to be able to start.
```

Comment:
When starting a workload pod that uses Longhorn volumes,
the Longhorn UI shows that the Longhorn volumes are attached quickly,
but it takes a long time for the volumes to finish mounting and for the workload to be able to start.

Reply:
Thank you. Fixed